src/components/SelectAssets/SelectAssetView.tsx

```diff
@@ -52,15 +33,7 @@ export const SelectAssetView = ({ onClick, toRoute, assetId }: SelectAssetViewPr
     <Switch location={location} key={location.key}>
       <Route
         path={SelectAssetRoutes.Search}
-        component={(props: RouteComponentProps) => (
-          <SelectAssets onClick={handleAssetSelect} {...props} />
-        )}
-      />
-      <Route
-        path={SelectAssetRoutes.Account}
-        component={(props: RouteComponentProps) => (
-          <SelectAccount onClick={handleAccountSelect} {...props} />
-        )}
+        component={(props: RouteComponentProps) => <SelectAssets onClick={onClick} {...props} />}
```

Comment:
We should add a back button to the `SelectAssets` modal, it's a bit jarring at the moment - when you select the asset from send modal, you can't get back to the send screen without selecting an asset.

<img width="514" alt="Screen Shot 2022-09-08 at 4 25 18 pm" src="https://user-images.githubusercontent.com/97164662/189049763-d301754d-4d23-415f-ad17-dbeea74e0642.png">

<img width="522" alt="Screen Shot 2022-09-08 at 4 22 53 pm" src="https://user-images.githubusercontent.com/97164662/189049883-acf559dd-d439-476f-b031-66232f602329.png">

Reply:
@reallybeard

Reply:
happy to take this on, any pointers on a proper example to follow so I don't color outside the lines?